### DISCUSSION TITLE
Automatically email/ book the house

### DISCUSSION THREAD

**Discussion by diyonshibu** [AnthonyBloomer/daftlistings#147] [General]
Is it possible to send an automated email to the real estate agent of a specific house if it meets the requirements. So basically it the Daft bot will automatically send a email from like a .txt file. Or even provide a clickable link in the terminal so you can do it manually

**Comment by sdfordham** [comment]
As discussed in #139, there is a captcha which means you cannot, however the discussion there suggests there's no captcha if you login (have not checked this). If that is the case then you could couple this library with [Selenium ](https://www.selenium.dev/) to achieve what you want.
